tests/test_fitresult.py

```diff
@@ -163,62 +168,75 @@ def test_correlation(minimizer_class_and_kwargs):
     assert pytest.approx(cor_mat[0, 1], rel=0.01) == cov_mat[0, 1]/(a_error * b_error)
     assert pytest.approx(cor_dict[(a, b)], rel=0.01) == cov_mat[0, 1]/(a_error * b_error)
 
+
 @pytest.mark.parametrize("minimizer_class_and_kwargs", minimizers)
-def test_errors(minimizer_class_and_kwargs):
+@pytest.mark.parametrize("sigma", [1, 2])
+def test_errors(minimizer_class_and_kwargs, sigma):
+    n_max_trials = 7  # how often to try to find a new minimum
     results = create_fitresult(minimizer_class_and_kwargs=minimizer_class_and_kwargs)
     result = results['result']
     a = results['a_param']
     b = results['b_param']
     c = results['c_param']
 
-    z_errors, new_result = result.errors(method="zfit_error")
-    if new_result is not None:
-        result = new_result
-    minos_errors, _ = result.errors(method="minuit_minos")
-
+    for n_trial in range(n_max_trials):
```

Comment:
I've added a loop that retries with the new result if a new minimum was found, otherwise the test would be "silently skipped". If you're ok with that, you can merge, the rest looks good!